alerting/src/main/kotlin/org/opensearch/alerting/resthandler/RestGetFindingsAction.kt

```diff
@@ -59,9 +57,7 @@ class RestGetFindingsAction : BaseRestHandler() {
 
         val getFindingsSearchRequest = GetFindingsRequest(
             findingID,
-            table,
-            severity,
-            detectionType
+            table
```

Comment:
we aren't resolving boolQueryBuilder param?

Reply:
we dpn't need to